### DISCUSSION TITLE
Question about the assignment

### DISCUSSION THREAD

**Discussion by natsuko68** [anyone-can-cook/rclass1#95] [Q&A]
① For question ４, from the small question 2, we need to type "typeof(), length(), and str()".
In that process, do we put the age separately? (eg: typeof(3), typeof(6)) If so, the answer shows all the same. 
Could you please teach me how to do this process?

②The files must be submitted in PDF and RMD.
I am now doing the questions separately on R, and typing the answers in a Word file.
In this case, is it possible to download the Word files in R studios to create PDF and RMD?(I've already complete for making an environment for Knit).

**Comment by upascal** [comment]
In response to 1. I believe the question asks to apply each of those three functions (1. typeof, 2. length, and 3. str) to the whole object "age", not each individual number within the list. 

2. I think you are suppose to write your answer underneath the text that says "ANSWER:" within the .rmd file, so when you knit the pdf together you get a single document with your code and with the answers.
